front-end/src/components/player-page/general.tsx

```diff
@@ -28,31 +35,35 @@ interface Props {
   };
 }
 
-function PlayerPageGeneral({games, loadGamesConfig, player, seasons}: Props)
+const StyledFlexBox = styled(Box)(({ theme }) => ({
+  'flex-wrap': 'wrap',
+  'column-gap': '16px',
+  'justify-content': 'space-between',
+  'display': 'flex',
+  'row-gap': '16px',
+  'align-content': 'flex-start',
+}));
+
+function PlayerPageGeneral({games, loadGamesConfig, player, seasons, championData, playerStats}: Props)
 {
-  if (!games)
-  {
-    games = [];
-  }
+  games = games ?? [];
+  const results = {wins: 0, losses: 0};
+  playerStats.map(cur => results[cur.win ? 'wins' : 'losses'] += 1, );
   return (
     <Box>
-      <Grid container>
-        <Grid item xs={12} md={4}>
-          <Container sx={{pr: 0, pl: 0}}>
-            <GamesFilter sx={{mb: 2}}
-              useSeason={true}
-              seasonConfig={{...loadGamesConfig.seasonConfig, seasons: seasons.filter(season => season.active)}}
-              useRole={true} roleConfig={loadGamesConfig.roleConfig} />
-            <RankFlag sx={{mb: 2}} player={player}></RankFlag>
-            <Hidden smDown>
-              <PlayerRadarCard sx={{mb: 2}} games={games}></PlayerRadarCard>
-            </Hidden>
-          </Container>
-        </Grid>
-        <Grid item xs={12} md={8}>
-          <GameSummaryList gameList={games} loadGamesConfig={loadGamesConfig} seasons={seasons}></GameSummaryList>
-        </Grid>
-      </Grid>
+      <FilterBar
+        seasonConfig={{...loadGamesConfig.seasonConfig, seasons: seasons.filter(season => season.active)}}
+        roleConfig={loadGamesConfig.roleConfig} />
+      <Box sx={{display: 'flex'}}>
+          <StyledFlexBox>
+            <RankFlag sx={{ minWidth: '170px', flexGrow: 1}} player={player}></RankFlag>
+            <WinRateBox hasData={results.wins > 0 || results.losses > 0} {...results} />
+            <ChampionOverview championData={championData}  sx={{width: '100%'}}/>
+            {/* <PlayedSeasons playerPuuid={player?.puuid} setSeason={loadGamesConfig.seasonConfig.setSeasonId} allSeasons={seasons} /> */}
```

Comment:
Commented out code

Reply:
Imma keep this there. Because its an inproress thing. I want to add it just figuring out how to do it still.